### DISCUSSION TITLE
New version permissions.. What are they explicitly?

### DISCUSSION THREAD

**Discussion by danSpotter** [kheina-com/Blue-Blocker#154] [Q&A] [answered]
I discovered the extension was disabled and when I went to reenable it I was met with this pop-up. 

Read and change your data on all twitter.com sites and twitter.com
Manage your apps, extensions and themes.

These are pretty vague descriptions from google. What can the new version do?

**Comment by sodablue** [comment]
Looking at the changes in github I see now some mention of a soupcan extension.   Is this why?   Is it necessary?

**Comment by kheina** [comment, marked as answer]
hey all, I understand the concerns and I'm working on removing the new management requirement now. long story short, they're needed to detect if another extension is installed and display the option to enable integration with that extension. currently only soupcan, but I've heard about others that want to add integrations and I wanted to enable that.

I've since removed the management permission and use a different way to detect if the extension exists or not and will be updated in v0.3.1. for the moment I'm going to close this discussion in favor of https://github.com/kheina-com/Blue-Blocker/issues/155 which will track the removal of this requirement.

edit: the management requirement has been removed and all stores have been updated as of last night. you can view the issue above for more detail
